crates/cxx-qt-gen/src/generator/cpp/constructor.rs

```diff
@@ -20,6 +20,7 @@ fn default_constructor(
 ) -> GeneratedCppQObjectBlocks {
     GeneratedCppQObjectBlocks {
         methods: vec![CppFragment::Pair {
+            // TODO: if there is no qobject macro what do we do with the constructor?
```

Comment:
TODO what should happen to constructor on non QObject tagged classes? do we still do the same ?